README.adoc

```diff
@@ -313,7 +304,7 @@ Other options include `Debug` (disables optimizations) and `Coverage`.
 
 |`-DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON`
 |With CMake 3.9+, compiles the project with LTO (Link Time Optimization) enabled.
-Older CMake will only honor this option with the Intel compiler on Linux.
+Older versions of `CMake`` only honor this option with the Intel compiler on Linux.
```

Comment:
Not this line, but 'Dispatch' is used on line 296 
Stoopid github not letting me comment on unchanged lines... :(

Reply:
Oh, and line 310 as well....